master-docker-nonstandard-2/master.cfg

```diff
@@ -509,10 +510,15 @@ f_jepsen_mariadb.addStep(
     )
 )
 
-
 # Define a function to add test steps to a factory
 def add_test_steps(factory, test_types):
```

Comment:
You can add type hints to this as well. `FullTestConfig` and `BuildFactory`. See dataclass comment to see what `FullTestConfig` is.